src/plots/layout_attributes.js

```diff
@@ -224,5 +224,30 @@ module.exports = {
             'or a logo image, for example. To omit one of these items on the plot,',
             'make an item with matching `templateitemname` and `visible: false`.'
         ].join(' ')
+    },
+    modeBarStyle: {
```

Comment:
no :camel: in data/layout attributes!

Reply:
... but wait do we really want to make these layout attributes?

Reply:
Ha I see, @alexcjohnson , @nicolaskruchten and @jackparmer have agreed on making these layout attributes in https://github.com/plotly/plotly.js/issues/3063. Not much I can say I guess.

Reply:
Maybe you could chime in on how we should name those attributes. I didn't realize there no 🐫 in data/layout attributes! 😳

Reply:
I vote for @etpinard's suggestion. What do you guys think? @alexcjohnson @nicolaskruchten @jackparmer